vcpkg-configuration.json

```diff
@@ -1,5 +1,4 @@
 {
-  "overlay-ports": [],
```

Comment:
I put this there for reference if we need it in the future. Then we don't have to look it up in the docs.
You can remove it if you prefer, but personally, I'd leave it.

Reply:
The vcpkg tool removed it

Reply:
Interesting. I don't really care, leave it as is.